PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -193,6 +223,21 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
   o2::framework::OutputObj<ZorroSummary> zorroSummary{"zorroSummary"};
   int currentRun{-1};
 
+  /// Set standard preselection gap trigger (values taken from UD group)
+  void setSGCutsFromUD(SGCutParHolder& sgCuts)
```

Comment:
"from UD" is not an attribute that should appear in the name.

Reply:
fixed